trieste/models/gpflux/interface.py

```diff
@@ -88,14 +88,3 @@ def get_observation_noise(self) -> TensorType:
             raise NotImplementedError(f"Model {self!r} does not have scalar observation noise")
 
         return noise_variance
-
-    def __deepcopy__(self, memo: dict[int, object]) -> GPfluxPredictor:
```

Comment:
GPflux now works? some recent changes on GPflux side?

Reply:
No GPflux still fails, but I've moved the exception with the useful advice about using track_state=False to the optimize method itself, so it'll catch any model that fails to deepcopy, not just our ones.

Reply:
ah, right, that's indeed better